packages/libs/multi-blast/config-overrides.js

```diff
@@ -10,18 +10,37 @@ module.exports = function override(config, env) {
     ...config,
     resolve: {
       ...config.resolve,
-      modules: [
-        path.join(__dirname, 'node_modules'),
-        ...(config.resolve.modules || ['node_modules']),
-      ],
+      fallback: {
+        path: 'path-browserify',
+        stream: 'stream-browserify',
+        querystring: 'querystring-es3',
+        assert: 'assert',
+        buffer: 'buffer',
+        fs: false,
+      },
     },
-    resolveLoader: {
-      ...config.resolveLoader,
-      modules: [
-        path.join(__dirname, 'node_modules'),
-        ...(config.resolveLoader.modules || ['node_modules']),
+    plugins: [
+      ...config.plugins,
+      new webpack.ProvidePlugin({
+        // This is needed by shape2geohash, used by MapVEuMap
+        process: 'process/browser',
```

Comment:
Does multi-blast use shape2geohash? I'd guess it doesn't. Does something else need the `process` global JS variable?

Reply:
I just copied that in an attempt to get the local dev server for multi-blast working. It can probably be removed.

Reply:
I couldn't get the local dev server working with qa or feature back ends.

Following https://github.com/VEuPathDB/web-monorepo/tree/main/packages/libs/multi-blast README I set all but WDK_AUTH_KEY - which I wasn't sure about (I thought we haven't used this for years?)

Should I have been able to get it working? @dmfalke

Reply:
No, I wasn't able to get it working, either. I've been using a local ortho-site using dfalke-b.orthomcl.org as a backend. I will try to get it working, maybe after the main work is complete.